### DISCUSSION TITLE
[需求] 帮我看看规则问题出在哪里

### DISCUSSION THREAD

**Discussion by DuJiaHui123** [aooiuu/any-reader#170] [Q&A]
### 功能描述

{
    "id": "6839b9ac-5d5c-47bc-8cff-37775bf95410",
    "createTime": 1721717026127,
    "modifiedTime": 1721717026127,
    "enableUpload": true,
    "author": "",
    "loadJs": "",
    "cookies": "",
    "name": "测试用",
    "host": "http://www.abcbiquge.com",
    "icon": "",
    "group": "",
    "contentType": 1,
    "sort": 100047,
    "useCryptoJS": false,
    "userAgent": "",
    "enableDiscover": false,
    "discoverUrl": "",
    "discoverNextUrl": "",
    "discoverList": "",
    "discoverTags": "",
    "discoverName": "",
    "discoverCover": "",
    "discoverChapter": "",
    "discoverDescription": "",
    "discoverResult": "",
    "enableSearch": true,
    "searchUrl": "/modules/article/search.php?searchkey=$keyword",
    "searchAuthor": "//td[3]/a/text()",
    "chapterCover": "",
    "chapterTime": "",
    "discoverAuthor": "",
    "searchList": "//table[@class=\"grid\"]/tbody/tr[position()>1]",
    "searchTags": "",
    "searchName": "//td[1]/a/text()",
    "searchCover": "",
    "searchChapter": "//td[2]/a/text()",
    "searchDescription": "",
    "searchResult": "//td[1]/a/@href",
    "enableMultiRoads": false,
    "chapterRoads": "",
    "chapterRoadName": "",
    "chapterUrl": "",
    "chapterNextUrl": "",
    "chapterList": ".box_con > div > dl >dd > a",
    "chapterName": "text",
    "chapterResult": "href",
    "contentUrl": "",
    "contentNextUrl": "",
    "contentItems": "#content@html",
    "contentDecoder": "",
    "viewStyle": 0,
    "discoverItems": ""
}

**Comment by DuJiaHui123** [comment]
搜索封面图片没有出来以及点进去没有章节

**Comment by aooiuu** [comment]
1. 没有章节替换

```json
    "searchName": "td:nth-child(1)@text",
    "searchResult": "td:nth-child(1) a@href",
```

2. 没有封面需要补充 `searchCover`

**Comment by aooiuu** [comment]
xpath 支持可能有问题
